disemvowel/solution.js

```diff
@@ -1,3 +1,3 @@
 function disemvowel(str) {
-  return str.replace(/[aAeEiIoOuU]/g,'');
+  return str.replace(/[aeiou]/ig,'');
```

Comment:
Workflow feedback: I notice that you also have a file changed from a previous assignment here--preferred style is to fix your old code on a different branch, and make sure this branch only has changes about the current assignment.